content/departments/support/tools/support-logging-guide.md

```diff
@@ -0,0 +1,189 @@
+- Does the customer have a mono-repo, and how large? Do they have smaller repos, how many? Or do they have both?
+- How severe are the log level warnings, and how often? `warn` is less severe than `crit`
+- Are there issues connecting to one code host and not another?
+
+Look for clues in the logs that a service can succesfully send and receive requests. Which service was sending a request and where was it to? What is the http response back? Is there a response at all? That will guide you towards the root cause.
+
+#### Gathering logs and their levels
+
+Logs are very useful in the right situations! Having an intuition for when to ask for them builds over time, and ideally you would have them and have attempted to interpret them before escalating a case. TODO add link to RFH stuff
+So before asking a customer for logs, consider a few things:
+
+Urgency of the request...
+
+- If Sourcegraph's frontend is not starting and the web UI is inaccessable, yes you likely need logs!
+
+Less urgent...
+
+- Not every user you talk to has the ability to gather logs. Was the case reported by someone that appears to be a **site administrator**?
+- Gathering logs and sharing them with AER's takes customer time and effort. Does it sound like logs are the only place to find the answer? Or are there other places to check like Grafana or in the Sourcegraph site-admin pages? TODO Link to this section ### Site administrators
+
+Blockers to sharing logs...
+
+- Sometimes customer security restrictions dictate what they can and cannot share. Airgapped customers are more likely to restrict what information they can share. In cases like this, can the customer be guided to which services to check in the logs or observability pages? Consider testing out commands in one of our TODO add AER test instance doc or local instance.
+
+Reported symptoms do not sound like Sourcegraph is 'broken'...
+
+- Is the question more of a "how do I..." type question, such as "This query ... is not returning any results, what am I doing wrong?" Or "my batch spec is failing with this error message..." Consider asking for more details on what they are attempting and testing it before asking for logs.
+
+##### Accessing logs
+
+The way to access logs depends on the deployment method and which service you need logs for. For quick reference see the [Support command generator tool](https://sourcegraph.github.io/support-generator/)
+
+TODO: Add more specifics on commands to gather logs by deployment type, etc
+
+[Logging levels are set as an environmental variable `SRC_LOG_LEVEL`.](https://docs.sourcegraph.com/admin/observability/logs) The default logging level in cluster deployments is `dbug`, and in docker it is `warn`. **Site administrators** may have changed the logging level to _more_ or _less_ verbose. The most verbose logging level is `dbug` and the least is `crit`.
```

Comment:
TODO: HOW to set the log level

Reply:
I linked this doc https://docs.sourcegraph.com/admin/install/docker-compose/operations#set-environment-variables